eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -270,7 +270,7 @@ stages:
               -repoName "$(SdkRepoOwner)/$(SpecRepoName)" `
               -pullRequestNumber "$(System.PullRequest.PullRequestNumber)" 
           displayName: 'Detect API Change and Create API Review'
-          condition: and(succeeded(), ne(variables['SpecGenSdkArtifactPath'], ''), ne(variables['SpecGenSdkArtifactName'], ''))
+          condition: and(succeeded(), ne(variables['SpecGenSdkArtifactPath'], ''), ne(variables['SpecGenSdkArtifactName'], ''), eq(variables['HasAPIViewArtifact'], 'true'))
```

Comment:
Wouldn't the other variables be enough? I don't think they are set unless there is an artifact.

Reply:
No, there is a JSON artifact that contains metadata about the other artifacts. In some cases that might be all that is present.

Reply:
Ok, can we simplify this condition to only look at the HasAPIViewArtifact then?